src/command.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
figure out what to do with this license header

Reply:
@weichweich Remove it?

Reply:
> @weichweich Remove it?

I guess GPL doesn't allow that. Just keep it. We may be able to add our copyright to the one of parity?

```
 // Copyright 2017-2020 Parity Technologies (UK) Ltd. 
 // Copyright 2019-2020 Botlabs GmbH
```

Or something like that...

Reply:
I like that. We should check with @GustavHemmelmayr.

Reply:
well, comirights for two companies for same software is confusing unless we really programmed together - stuff that is done by us can be labled for BOTLabs GmbH (please spell with the first three letters as capital letters) and stuff that is done by Parity is labeld by them and we just refer to it, right?

GPL does allow a reference to the company, please always use the original wording in the license part of the software.